src/granite_orm/fields.cr

```diff
@@ -1,6 +1,8 @@
 require "json"
 
 module Granite::ORM::Fields
+  alias Type = Bool | Float32 | Float64 | Int32 | Int64 | String | Time | Nil | Hash(String, JSON::Type) | Array(JSON::Type)
+
```

Comment:
Here are the DB::Types: https://github.com/crystal-lang/crystal-db/blob/master/src/db.cr#L72

[Nil, String, Bool, Int32, Int64, Float32, Float64, Time, Bytes]

If we add `Bytes`, we can remove the `DB::Any` or if we leave the DB::Any in, we can remove `Bool | Float32 | Float64 | Int32 | Int64 | String | Time | Nil`

Reply:
Type alias looks like this now `alias Type = DB::Any | Hash(String, JSON::Type) | Array(JSON::Type)`